server/screeningsList.js

```diff
@@ -0,0 +1,43 @@
+import fetch from 'node-fetch';
+
+export const getScreeningsWithMovies = {
+    loadData : async () => {
+        const res = await fetch("https://plankton-app-xhkom.ondigitalocean.app/api/screenings?populate=movie");
+        const data = await res.json();
+
+        return data;
+    }
+}
+// Formatting and filtering JSON to send only relevant data for rendering. 
+export const getScreeningsList = async (apiHandler) => {
+        
+    const data = await apiHandler.loadData();
+    
+    const res = data.data
+    
```

Comment:
Blank lines like these confuse me, because they make it less apparent that the next line is actually chained on the previous one.

In situations like this, I would just trust some auto-formatting software, e.g. Prettier, to style the code, which I believe would look different than this.